site2/docs/io-quickstart.md

```diff
@@ -473,18 +460,22 @@ This example uses the PostgreSQL 12 docker image to start a single-node PostgreS
    2020-05-11 20:09:24.533 UTC [1] LOG:  database system is ready to accept connections
    ```
 
-4. Access to PostgreSQL.
+4. Access to PostgreSQL container.
 
    ```bash
    docker exec -it pulsar-postgres /bin/bash
    ```
 
-5. Create a PostgreSQL table _pulsar_postgres_jdbc_sink_.
+5. Login PostgreSQL with default username and password:
 
    ```bash
    psql -U postgres postgres
+   ```
+
+6. Input following commands to create `pulsar_postgres_jdbc_sink` table:
```

Comment:
```suggestion
6. Create a `pulsar_postgres_jdbc_sink` table using the following command:
```

1. Show "action / purpose" first.
2. Below is one command, so remove "s" for "command".

Reply:
Got it, thanks.